.github/workflows/builds.yml

```diff
@@ -68,6 +68,8 @@ jobs:
         PLATFORM:
           - NAME: "CUDA"
             CONTAINER: "ghcr.io/acts-project/ubuntu1804_cuda:v11"
+          - NAME: "CUDA"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:v13"
```

Comment:
Would it maybe be worth it just replacing the previous CUDA platform? It might be confusing to have two CUDA platforms, and also it takes up more CI time.

Reply:
:thinking: I'd prefer to keep CUDA 10.2 compatibility alive for a little longer. With the amount of CI jobs that we run these days, I think we should be able to afford this...

Reply:
Sure, no problem. What do you think of naming one "CUDA 10.2" and the other "CUDA 11.3"? Might make it easier to see which of the CI jobs breaks.

Reply:
Nevermind, looking at it that would break some of the CI logic. Let's just go with this.